Library/Formula/getxbook.rb

```diff
@@ -1,11 +1,21 @@
-require 'formula'
-
 class Getxbook < Formula
-  homepage 'http://njw.me.uk/software/getxbook/'
-  url 'http://njw.me.uk/software/getxbook/getxbook-1.1.tar.bz2'
-  sha1 '9d47918ed77e8fb4f4e8b3c412cdcc82834be3e8'
+  homepage "http://njw.me.uk/software/getxbook/"
+  url "http://njw.me.uk/software/getxbook/getxbook-1.1.tar.bz2"
+  sha1 "9d47918ed77e8fb4f4e8b3c412cdcc82834be3e8"
+
+  option "with-gui", "Build the GUI"
+
+  depends_on "tcl-tk" if build.with? "gui"
 
   def install
-    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}", "install"
+    args = %W[CC=#{ENV.cc} PREFIX=#{prefix}]
+    args << "install" if build.with?("gui")
+
+    system "make", *args
+    bin.install "getgbook", "getabook", "getbnbook"
```

Comment:
So if I install `with-gui`, will I get two copies of binary, one in `bin`, one in app bundle? If that's the case, using symlink may be a better idea.